File: webapp/src/main/resources/static/controller/ExaminationNumberBookingController.js
'use strict';
/*******************************************************************************
 * ExaminationNumberBookingController
 ******************************************************************************/
app.controller('ExaminationNumberBookingController', ['ExaminationNumberBookingService', '$scope', function (ExaminationNumberBookingService, $scope) {
  
  $scope.init = function () {
    console.log('ExaminationNumberBookingController.js: init()');
    ExaminationNumberBookingService.init();
    $scope.serviceName = ExaminationNumberBookingService.getInitResult()[0].message;
  }
  
}]);

File: management/src/main/java/io/hexaforce/management/controller/ExaminationNumberBookingController.java
package io.hexaforce.management.controller;

import java.util.Arrays;
import java.util.List;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.http.HttpStatus;
import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.ResponseBody;
import org.springframework.web.bind.annotation.RestController;

import io.hexaforce.dievas.commons.DievasServerMesseage;
import io.hexaforce.dievas.service.ExaminationNumberBookingService;
import lombok.extern.slf4j.Slf4j;

/**
 * ExaminationNumberBooking router function.
 * 
 * @version 1.0.0.BUILD-SNAPSHOT
 * @author T.Tantaka 
 */

@Slf4j
@RestController
@RequestMapping("/api")
public class ExaminationNumberBookingController {
	
	@Autowired
	private ExaminationNumberBookingService examinationNumberBookingService;
	
	@ResponseBody
	@GetMapping(value = "/examination-number-booking")
	public ResponseEntity<List<DievasServerMesseage>> init() {
		log.info("Controller(java): init()");
		String x = examinationNumberBookingService.init();
		return new ResponseEntity<List<DievasServerMesseage>>(Arrays.asList(new DievasServerMesseage(x)),HttpStatus.OK);
	}
	
}